packages/wp-story-editor/src/api/storyLock.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { addQueryArgs } from '@web-stories-wp/design-system';
+
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+export function getStoryLockById(storyId, stories) {
+  const path = addQueryArgs(`${stories}${storyId}/lock`, {
+    _embed: 'author',
+  });
+
+  return apiFetch({ path });
+}
+
+export function setStoryLockById(storyId, stories) {
+  const path = `${stories}${storyId}/lock`;
```

Comment:
Realized that this trailing slash is missing here (even before). Let's add it. We've had some WAF compatibility issues in the past when the trailing slashes were omitted.

```suggestion
  const path = `${stories}${storyId}/lock/`;
```

Reply:
Done in https://github.com/google/web-stories-wp/pull/8966/commits/a332ebafd61dfc4a7f9ceddb84412e739ce7b117